.github/styles/Vocab/Python/accept.txt

```diff
@@ -0,0 +1,2 @@
+wsgi
+middleware
```

Comment:
can we configure this to use the same wordlist we use for sphinx spell checking?